website/README.md

```diff
@@ -0,0 +1,41 @@
+# Deno Documentation Parser
+
+As a modern TypeScript run-time, Deno aims to have an online documentation website
+which enables JavaScript community to see their code documentation by providing URL
+of the source code. (like godoc)
+
+This subdirectory at Deno is to maintain such functionality.
+
+# Current plan
+By using TypeScript API, we can parse a file to its syntax tree and then extract
+the useful information from there, but dealing directly with AST and trying to
+convert it to HTML pages might complicate things. Therefore we have to separate
+the parser and the renderer.
+
+## Step 1) Parse TS/JS AST
+In this step, we try to parse an AST into the simplest possible object that contains
+all the useful data which we will use to render the HTML pages.
+The result of this step is an array of objects, where each of those must have a
+`type` field.  
+In most cases, `type` corresponds to the `kind` value from the parsed AST.
```

Comment:
Maybe you can give a little example of what this array of objects might look like? Maybe some pointers of where to look in the code?

Reply:
I will add `doc.d.ts` in a follow-up pr, and I will point to that whenever it gets ready

Reply:
Ok - np.